projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js

```diff
@@ -35,17 +35,17 @@ function getReachForAccessLevelKey(
 	paidSubscribers,
 	socialFollowers
 ) {
-	if ( emailSubscribers === null || paidSubscribers === null || socialFollowers === null ) {
+	if ( ! socialFollowers ) {
```

Comment:
Interesting, I suspect we could remove `socialFollowers` since it's not used in the fn?

Reply:
I agree.

Reply:
Taking it out affects the behavior.  Not sure why.